Adds overlay list paths specific to host xternal.mine.nu.

Index: layman/layman.cfg
===================================================================
--- layman.orig/layman.cfg
+++ layman/layman.cfg
@@ -32,6 +32,9 @@ make_conf : %(storage)s/make.conf
 #            file:///usr/portage/local/layman/my-list.xml
 
 overlays  : http://www.gentoo.org/proj/en/overlays/layman-global.txt
+            file:///srv/www/rpo.xw.ath.cx/htdocs/gentoo/portage/overlays/xworld.xml
+            file:///srv/www/rpo.xw.ath.cx/htdocs/gentoo/portage/overlays/hosted.xml
+            file:///srv/www/rpo.xw.ath.cx/htdocs/gentoo/portage/overlays/foreign.xml
 
 #-----------------------------------------------------------
 # Proxy support
